src/main/java/nl/novi/techiteasy/dtos/TelevisionDto.java

```diff
@@ -0,0 +1,180 @@
+    public Boolean wifi;
+    public Boolean voiceControl;
+    public Boolean hdr;
+    public Boolean bluetooth;
+    public Boolean ambiLight;
+    public Integer originalStock;
+    public Integer sold;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public Double getPrice() {
+        return price;
+    }
+
+    public void setPrice(Double price) {
+        this.price = price;
+    }
+
+    public Double getAvailableSize() {
+        return availableSize;
+    }
+
+    public void setAvailableSize(Double availableSize) {
+        this.availableSize = availableSize;
+    }
+
+    public Double getRefreshRate() {
+        return refreshRate;
+    }
+
+    public void setRefreshRate(Double refreshRate) {
+        this.refreshRate = refreshRate;
+    }
+
+    public String getScreenType() {
+        return screenType;
+    }
+
+    public void setScreenType(String screenType) {
+        this.screenType = screenType;
+    }
+
+    public String getScreenQuality() {
+        return screenQuality;
+    }
+
+    public void setScreenQuality(String screenQuality) {
+        this.screenQuality = screenQuality;
+    }
+
+    public Boolean getSmartTv() {
+        return smartTv;
+    }
+
+    public void setSmartTv(Boolean smartTv) {
+        this.smartTv = smartTv;
+    }
+
+    public Boolean getWifi() {
+        return wifi;
+    }
+
+    public void setWifi(Boolean wifi) {
+        this.wifi = wifi;
+    }
+
+    public Boolean getVoiceControl() {
+        return voiceControl;
+    }
+
+    public void setVoiceControl(Boolean voiceControl) {
+        this.voiceControl = voiceControl;
+    }
+
+    public Boolean getHdr() {
+        return hdr;
+    }
+
+    public void setHdr(Boolean hdr) {
+        this.hdr = hdr;
+    }
+
+    public Boolean getBluetooth() {
+        return bluetooth;
+    }
+
+    public void setBluetooth(Boolean bluetooth) {
+        this.bluetooth = bluetooth;
+    }
+
+    public Boolean getAmbiLight() {
+        return ambiLight;
+    }
+
+    public void setAmbiLight(Boolean ambiLight) {
+        this.ambiLight = ambiLight;
+    }
+
+    public Integer getOriginalStock() {
+        return originalStock;
+    }
+
+    public void setOriginalStock(Integer originalStock) {
+        this.originalStock = originalStock;
+    }
+
+    public Integer getSold() {
+        return sold;
+    }
+
+    public void setSold(Integer sold) {
+        this.sold = sold;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
alle jouw logica is beter te plaatsen in Service. In de DTO moeten lieve alleen maar variabele worden gedeclareerd, constructor, getters & setters aangemaakt